swift-idl-parser/src/main/java/com/facebook/swift/parser/model/TypeAnnotation.java

```diff
@@ -32,7 +32,7 @@
     public TypeAnnotation(String name, String value)
     {
         this.name = checkNotNull(name, "name");
-        this.value = Preconditions.checkNotNull(value, "value");
+        this.value = value;
```

Comment:
I checked usage of TypeAnnotation and didn't find any.  I think the null-ness of value depends on name but we can't enforce that here.